[Workflow] Add note about required packages (#13565)
symfony/symfony-docs: workflow.rst

            ],
        ]);

.. note::

>     If `symfony/security-core` package is not installed, these methods are not available:

Reviewer: These expressions instead of methods, wdty?

Author: I changed `methods` to `expressions`, please take a look

            ],
        ]);

.. note::

>     `symfony/expression-language` and `symfony/security-core` is required to use `guard` configuration option.

Reviewer 2: ```suggestion
    ``symfony/expression-language`` and ``symfony/security-core`` packages are required to
    use ``guard`` configuration option.
```

.. tip::

    These expressions are available by default:

    * ``subject``

>     These expressions are available if `symfony/security-bundle` is installed:

Reviewer 2: ```suggestion
    These expressions are available if ``symfony/security-bundle`` is installed:
```

    * ``is_anonymous``
    * ``is_authenticated``
    * ``is_fully_authenticated``
    * ``is_granted``
    * ``is_remember_me``

>     These expressions are available if `symfony/validator` is installed:

Reviewer 2: ```suggestion
    These expressions are available if ``symfony/validator`` is installed:
```
